cpp_exercise/src/lib/ipcmsgqlib.cc

```diff
@@ -11,23 +11,36 @@
 
 MsgQueueSender::MsgQueueSender(const std::string & filename): _file_name(filename){
 
-        std::cout<<"**************************************************************"<<std::endl;
-        std::cout<<"****************************IPCSender: MESSAGE QUEUE PROTOCOL**************************"<<std::endl;
-        std::cout<<"**************************************************************"<<std::endl;
-        std::cout<<"**************************************************************"<<std::endl;
 
+  
         //removing and unlinking from old shared memories
 
-        remove(_msg_queue_name.c_str());
-        ret = mq_unlink(_msg_queue_name.c_str());
+
+        remove(msg_queue_name.c_str());
+        ret = mq_unlink(msg_queue_name.c_str());
+
```

Comment:
I don't think you need remove() here, mq_unlink() will remove the message queue name

Reply:
Fixed